src/sentry/static/sentry/app/actionCreators/tags.jsx

```diff
@@ -75,6 +75,23 @@ export function fetchProjectTags(orgId, projectId) {
   return promise;
 }
 
+/**
+ * Fetch tag values for a single project. Used for sentry9 views.
+ */
+export function fetchProjectTagValues(api, orgId, projectId, tagKey, search = null) {
+  const url = `/projects/${orgId}/${projectId}/tags/${tagKey}/values/`;
+
+  const query = {};
+  if (search) {
+    query.query = search;
+  }
+
+  return api.requestPromise(url, {
+    method: 'GET',
+    query,
+  });
+}
```

Comment:
Should this trigger some sort of `action`?

Reply:
We've been moving API calls into actionCreators and returning promises as a first step towards separating rendering logic and API calls. A follow up to that could be to leverage redux stores more.